### PR TITLE
PR_0x02-Frontispiece.md

### PR DESCRIPTION
**Pull request by kunwar-atul-hax0r**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [ ] Your PR is linked to an issue.
- [ ] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [ ] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [ ] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

This PR covers issue #< insert number here >.

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

**Comment by commjoen** [comment]
https://github.com/OWASP/owasp-masvs/pull/426/checks?check_run_id=567407425 seems to be failing. Can you please fix the markdown issues?

**Comment by kunwar-atul-hax0r** [comment]
All the necessary changes have been made.

File: Document-hi/0x02-Frontispiece.md [hunk 1]
```diff
@@ -41,16 +60,39 @@ Copyright © 2020 The OWASP Foundation.This work is licensed under a [Creative C
...
+| --- | ------------------  |
+| चीनी (पारंपरिक)| पीटर ची, और लेक्स चिएन, हेनरी हू, लियो वांग|
+| चीनी (सरलीकृत)| बॉब पेंग, हेरोल्ड ज़ैंग, जैक एस|
+| फ्रेंच | रोमुआल्ड स्ज़ुडलारेक, एबदर्रहमान आफ़ताही, क्रिश्चियन डोंग (समीक्षा)|
+| जर्मन | रोक्को ग्रिट्ज़, स्वेन शेलियर (समीक्षा)|
+| जापानी | कोकी तकेयमा, रियोतरो ऑकदा (समीक्षा)|
+| कोरिअन| यौन्ग्जये जेओं, जेओंग्वों चो, जियौ हान, जियेऑन सुन्ग|
+| रूसी | गल्ल मक्सिम, एउगेन मर्त्य्नोव, चलनोकोव व्लादिस्लाव (समीक्षा), ओप्री ईगोर (समीक्षा), टेरेशिन दिमित्री (समीक्षा)|
+|स्पेनिश | मार्टिन मार्सिकनो, कार्लोस होलगुएरा|
+<br/>
```
Comment (sushi2k) [inline Document-hi/0x02-Frontispiece.md:73]: Please also add Hindi as another language and everybody who worked on it.

Comment (kunwar-atul-hax0r) [reply Document-hi/0x02-Frontispiece.md:73]: Sure.

File: Document-hi/0x02-Frontispiece.md [hunk 2]
```diff
@@ -1,31 +1,50 @@
 # About the Standard
 
+## मापदंड के बारे में
+
```
Comment (gitmukeshsharma) [inline Document-hi/0x02-Frontispiece.md:4]: # मानक के बारे में

**Comment by kunwar-atul-hax0r** [comment]
@gitmukeshsharma updated the suggestion.

**Comment by gitmukeshsharma** [comment]
I have reviewed the file . @sushi2k could you please merge it.

**Comment by sushi2k** [comment]
@kunwar-atul-hax0r  can you pleaser remove the english original sentences, so we only have the hindi translation? Thanks

**Comment by kunwar-atul-hax0r** [comment]
Hi @sushi2k I have removed the English sentences as per your request.

**Pull request merged**